bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -325,6 +329,7 @@ def test_filter_pushdown(datapath, file_path, op):
         check_pandas_types=False,
         sort_output=True,
         reset_index=False,
+        check_names=False,
```

Comment:
Why is this changed? How would this PR impact output names?

Reply:
Was necessary with previous version of code but not the re-worked code so I've reverted this.